.github/workflows/review-pr.yml

```diff
@@ -0,0 +1,70 @@
+name: Use OpenDevin to Review Pull Request
+
+on:
+  pull_request:
+    types: [synchronize, labeled]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  open-devin:
+    if: contains(github.event.pull_request.labels.*.name, 'dogfood-this')
```

Comment:
Consider using a different label, perhaps `review-this`
so it's distinct from the solve-issue workflow?

( Yup, the other workflow could use a better label too :D )

Reply:
Agreed. `dogfood` is not wrong but isn't concise enough.